src/components/label/Label.js

```diff
@@ -52,7 +61,7 @@ const Label = ({ text, tooltip, infoTooltip, infoIconProps, showOptionalText, hi
     }
 
     return (
-        <StandardLabel labelContent={labelContent} tooltip={tooltip}>
+        <StandardLabel labelContent={labelContent} tooltip={tooltip} {...rest}>
```

Comment:
It looks like all the changes to `Label` can be reverted, I think?

Reply:
done